bin/get-pgmonitor.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+# Copyright 2017 - 2020 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Getting pgMonitor..."
+PGMONITOR_COMMIT='v4.4-RC2'
```

Comment:
```suggestion
PGMONITOR_COMMIT='v4.4-RC3'
```
^ is now available. Do we need to make any other updates for this?

Reply:
Nothing to this PR as it stands. It looks like the only update is to the prometheus.yml file. At some point, we'll need to integrate that to our build process in place of the copy we're maintaining. If you like, I can add that as another commit. It would basically replace this
https://github.com/CrunchyData/postgres-operator/blob/master/installers/metrics/ansible/roles/pgo-metrics/files/prometheus.yml
with
https://github.com/CrunchyData/pgmonitor/blob/master/prometheus/crunchy-prometheus.yml.containers

Reply:
I actually have that change staged to push up in my pr for alertmanager. I deleted `prometheus.yml` and updated the installer to copy it into place from the downloaded pgmonitor repo files. I was handling the merge conflict where that files was updated and will push up the changes soon.

Reply:
Ok, great, thank you @jmckulk. I pushed the updated to RC3, so that should be all for this PR then.